scenes/world_map/teleporter.gd

```diff
@@ -12,13 +12,21 @@ const SPAWN_POINT_GROUP_NAME: String = "spawn_point"
 		_update_available_spawn_points()
 		notify_property_list_changed()
 
+@export var use_transition: bool = true:
+	set(new_val):
+		use_transition = new_val
+		notify_property_list_changed()
```

Comment:
I could be wrong, so let's leave it in for now, but I have a feeling that in practice this property will always be `true` when the destination is a different scene, and `false` when it's not.

Reply:
I was thinking there may be some cases in the same scene where you'd want to use a transition.